.bldr.toml

```diff
@@ -177,6 +177,14 @@ paths = [
   "components/pkg-export-docker/*"
 ]
 
+[hab-pkg-export-helm]
+plan_path = "components/pkg-export-helm"
+paths = [
+  "components/core/*",
+  "components/common/*",
+  "components/pkg-export-kubernetes/*"
```

Comment:
I think we want to add `components/pkg-export-docker/*` here, too. If some file there changes, but nothing directly in `pkg-export-kubernetes`, then `pkg-export-kubernetes` will be built, but this won't.

Tracing up the dependency tree, it looks like we should also add `components/hab/*` for this and `[hab-pkg-export-docker]` as well.